src/region.js

```diff
@@ -284,19 +299,18 @@ const Region = MarionetteObject.extend({
     this._parentView.stopListening(view);
   },
 
-  _removeView(view, shouldDestroy) {
-    if (!shouldDestroy) {
-      this._detachView(view);
-      return;
-    }
-
+  destroyView(view) {
```

Comment:
```js
if (view._isDestroyed) {
  return;
}
```
I just started testing the related part of the next collection view and it has a near identical function.. https://github.com/marionettejs/backbone.marionette/pull/3244/files#diff-c4fcc4edadcafa174aa6de6124ecbeacR652

this can probably move to the mixin for v4